APCAppCore/APCAppCore/Library/APCTaskImporter.h

```diff
@@ -49,7 +49,7 @@
  
  See the .m file for technical notes on this method.
  */
-- (void) processTasks: (NSArray *) arrayOfTasks
+- (BOOL) processTasks: (NSArray *) arrayOfTasks
```

Comment:
If you are passing in an error pointer, they like for you to also return whether or not the method passed.